### PR TITLE
W-10703959: Create What's New article for Studio 7.12

### PR DESCRIPTION
**Pull request by fermujica**
Reference: [W-10703959](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00000ohxGJYAY/view)

Features listed for the What's New section for 7.12:

- Comprehensive Publication to Exchange
- MFA Support
- Comment and Uncomment Objects from the Canvas
- Improved Metadata Edition Experience
- Improved Design Center Import Experience

### CODE REVIEW

File: modules/ROOT/pages/whats-new-in-studio.adoc [hunk 1]
```diff
@@ -1 +1,31 @@
 = What’s New in Studio
+
+Anypoint Studio 7.12 introduces support for comprehensive publication of assets to Exchange and support for multi-factor authentication (MFA) to Anypoint Platform.
```
Comment (Crispy-Salesforce) [inline modules/ROOT/pages/whats-new-in-studio.adoc]: There seems to be a "extra" "support" verb in this sentence:
support for comprehensive publication of assets to Exchange and **_support_** for multi-factor authentication (MFA) to Anypoint Platform

Comment (Crispy-Salesforce) [inline modules/ROOT/pages/whats-new-in-studio.adoc]:
```suggestion
Anypoint Studio 7.12 introduces support for comprehensive publication of assets to Exchange and for multi-factor authentication (MFA) to Anypoint Platform.
```

File: modules/ROOT/pages/whats-new-in-studio.adoc [hunk 2]
```diff
@@ -1 +1,31 @@
...
+
+This version also improves the metadata edition and Design Center import experiences and adds a feature to comment and uncomment objects from the canvas.
+
+== Comprehensive Publication to Exchange
+
+When publishing an asset to Exchange, you can now add more information such as tags, categories, name, description, and documentation to your asset directly from Studio.
+
+== MFA Support
+
+You can now log in to Anypoint Platform using multi-factor authentication (MFA) from Studio's built-in browser.
```
Comment (Crispy-Salesforce) [inline modules/ROOT/pages/whats-new-in-studio.adoc]: Remove possessive case in objects. Replace by "The Studio built-in browser".

File: modules/ROOT/pages/whats-new-in-studio.adoc [hunk 3]
```diff
@@ -1 +1,31 @@
...
+
+== Comment and Uncomment Objects from the Canvas
+
+You can now comment and uncomment flows and components directly from the canvas:
+
+. Right-click the flow or component that you want to comment.
+. Select *Toggle Comment*.
++
+image::toggle-comment.png[]
```
Comment (Crispy-Salesforce) [inline modules/ROOT/pages/whats-new-in-studio.adoc]: Add ALT text to image.

File: modules/ROOT/pages/whats-new-in-studio.adoc [hunk 4]
```diff
@@ -1 +1,31 @@
...
+
+== Improved Metadata Edition Experience
+
+Studio 7.12 improves the overall performance of the metadata edition experience by reducing background processing and by asynchronously propagating the metadata through the Flow Reference component.
+
```
Comment (Crispy-Salesforce) [inline modules/ROOT/pages/whats-new-in-studio.adoc:27]: There are two blank lines between these two paragraphs. Is this intended?

File: modules/ROOT/pages/whats-new-in-studio.adoc [hunk 5]
```diff
@@ -1 +1,31 @@
 = What’s New in Studio
+
+Anypoint Studio 7.12 introduces support for comprehensive publication of assets to Exchange and for multi-factor authentication (MFA) to Anypoint Platform.
+
+This version also improves the metadata edition and Design Center import experiences and adds a feature to comment and uncomment objects from the canvas.
```
Comment (kbostrom) [inline modules/ROOT/pages/whats-new-in-studio.adoc]:
```suggestion
This version also improves the metadata edition and Design Center import experiences and adds a feature that enables you to comment and uncomment objects directly from the canvas.
```

File: modules/ROOT/pages/whats-new-in-studio.adoc [hunk 6]
```diff
@@ -1 +1,31 @@
...
+
+== Comprehensive Publication to Exchange
+
+When publishing an asset to Exchange, you can now add more information such as tags, categories, name, description, and documentation to your asset directly from Studio.
```
Comment (kbostrom) [inline modules/ROOT/pages/whats-new-in-studio.adoc]:
```suggestion
When publishing an asset to Exchange, you can now add more information (such as tags, categories, names, descriptions, and documentation) to your assets directly from Studio.
```
I added the parens just to lessen the distraction between "add information" and "to your assets."

File: modules/ROOT/pages/whats-new-in-studio.adoc [hunk 7]
```diff
@@ -1 +1,31 @@
...
+
+== MFA Support
+
+You can now log in to Anypoint Platform using multi-factor authentication (MFA) from the Studio built-in browser.
+
+== Comment and Uncomment Objects from the Canvas
```
Comment (kbostrom) [inline modules/ROOT/pages/whats-new-in-studio.adoc]:
```suggestion
== Comment and Uncomment Objects Directly from the Canvas
```
I kept struggling with whether the user was taking action *on* the canvas or *from* the canvas, but I chose to stick with your original wording.

File: modules/ROOT/pages/whats-new-in-studio.adoc [hunk 8]
```diff
@@ -1 +1,31 @@
...
+
+You can now comment and uncomment flows and components directly from the canvas:
+
+. Right-click the flow or component that you want to comment.
```
Comment (kbostrom) [inline modules/ROOT/pages/whats-new-in-studio.adoc]:
```suggestion
. Right-click the flow or component that you want to comment or uncomment.
```

File: modules/ROOT/pages/whats-new-in-studio.adoc [hunk 9]
```diff
@@ -1 +1,31 @@
...
+image::toggle-comment.png["Toggle comment highlighted when right-clicking the flow"]
+
+== Improved Metadata Edition Experience
+
+Studio 7.12 improves the overall performance of the metadata edition experience by reducing background processing and by asynchronously propagating the metadata through the Flow Reference component.
+
+
+== Improved Design Center Import Experience
+
+Studio 7.12 improves its import from Design Center experience by loading all REST APIs to import from Design Center simultaneously enabling you to search your desired API specification much faster.
```
Comment (kbostrom) [inline modules/ROOT/pages/whats-new-in-studio.adoc]:
```suggestion
Studio 7.12 improves its import from Design Center experience by loading all REST APIs to import from Design Center simultaneously, enabling you to search your desired API specification much faster.
```

**Review by kbostrom — DISMISSED** [review]
CE

**Review by Cristian-Venticinque — APPROVED** [review]

**Pull request merged**